types/chrome/index.d.ts

```diff
@@ -6902,7 +6912,6 @@ declare namespace chrome.runtime {
         storage?: {
             managed_schema: string;
         } | undefined;
-        system_indicator?: any;
```

Comment:
This API was never shipped in release browsers. [(It was a Chrome-only experiment which was later retracted.)](https://stackoverflow.com/questions/20306548/how-to-use-system-indicator-in-the-manisfet-of-a-chrome-packaged-app)